Makefile

```diff
@@ -100,10 +104,8 @@ publish: s3-release docker-release
 docker-publish: docker-build docker-push
 
 docker-build:
-	@if [ ! -f "./_tmp/clusterloader2" ]; then echo "downloading clusterloader2"; aws s3 cp --region us-west-2 s3://aws-k8s-tester-public/clusterloader2-linux-amd64 ./_tmp/clusterloader2; else echo "skipping downloading clusterloader2"; fi;
-	@if [ ! -d "${HOME}/go/src/k8s.io/perf-tests" ]; then echo "cloning perf-tests"; mkdir -p ${HOME}/go/src/k8s.io; pushd ${HOME}/go/src/k8s.io; git clone https://github.com/kubernetes/perf-tests.git; popd; else echo "skipping cloning perf-tests"; fi
-	@if [ ! -d "./_tmp/clusterloader2-testing-load" ]; then echo "copying clusterloader2/testing/load"; cp -rf ${HOME}/go/src/k8s.io/perf-tests/clusterloader2/testing/load ./_tmp/clusterloader2-testing-load; else echo "skipping copying clusterloader2/testing/load"; fi
-	docker build --network host -t $(AKT_IMG_NAME):$(AKT_TAG) --build-arg RELEASE_VERSION=$(AKT_TAG) .
+	curl -o kubectl -LO https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/${AKT_OS}/${AKT_ARCH}/kubectl
```

Comment:
could you move this to `hack/build.sh` (since that's what is invoked during the image build)? that way the result can be cached as an image layer versus downloading it every time.